docs/gl_objects/builds.rst

```diff
@@ -94,6 +94,16 @@ Full example with wait for finish::
         pipeline.refresh()
         time.sleep(1)
 
+You can trigger a pipeline using a token authentication instead of user
```

Comment:
```diff
- You can trigger a pipeline using a token authentication instead of user
+ You can trigger a pipeline using token authentication instead of user
```